packages/x-date-pickers/src/AdapterMoment/AdapterMoment.ts

```diff
@@ -472,7 +472,7 @@ export class AdapterMoment implements MuiPickersAdapter<Moment, string> {
   public getMeridiemText = (ampm: 'am' | 'pm') => {
     if (this.is12HourCycleInCurrentLocale()) {
       // AM/PM translation only possible in those who have 12 hour cycle in locale.
-      return this.moment
+      return defaultMoment
```

Comment:
Why using the default moment instead of the one provided by the user?

Reply:
Yes, I'm trying to only use the custom instance to create date, not for the meta data like locales